recipe/meta.yaml

```diff
@@ -38,20 +35,14 @@ outputs:
         - python
         - pip >=9.0.1
         - certifi >=2020.06.20
-        # freetype.pc requires zlib.pc
         - freetype
```

Comment:
please add here version constrain '2.10' to freetype dependency

Reply:
Is there a reason to not use 2.12, which seems to be available on all archs.